src/window.cpp

```diff
@@ -98,10 +101,10 @@ window_impl::window_impl(int pWidth, int pHeight, const char* pTitle,
         }
     }
 
-    // Set context (before glewInit())
+    /* Set context (before glewInit()) */
     MakeContextCurrent(this);
 
-    //GLEW Initialization - Must be done
+    /* GLEW Initialization - Must be done */
```

Comment:
If you are putting a comment *before* the code in the same line, it is even worse from a style point of view. Just use "//" for single line comments.